src/android/java/io/jxcore/node/BtToServerSocket.java

```diff
@@ -13,33 +13,39 @@
  */
 public class BtToServerSocket extends Thread implements StreamCopyingThread.CopyThreadCallback {
```

Comment:
A lot of my comments from BtToRequestSocket apply here since the code is so similar. This includes everything from never catching the Exception object to guarding start and stop via mStopped to making mHTTPPort into an accessor method to making sure we properly re-throw exceptions rather than silently swallow them, etc.